Remotion/Data/DomainObjects/DomainImplementation/Transport/XmlTransportItem.cs

```diff
@@ -215,7 +225,12 @@ private void SerializePropertyValue (XmlWriter writer, PropertyDefinition? prope
       }
     }
 
-     private static bool IsObjectID (Type valueType)
+    private bool IsDateOnlyType (Type type)
+    {
+      return type == typeof(DateOnly) || type == typeof(DateOnly?);
+    }
+
+    private static bool IsObjectID (Type valueType)
      {
        return typeof(ObjectID).IsAssignableFrom(valueType);
      }
```

Comment:
This methid has incorrect whitespace. please either fix entire method or revert the method-declaration line.